### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[href="\#"])

### ISSUE THREAD

**Issue by Umesh360480** [Umesh360480/AI#2]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Accessible University Demo Site - Inaccessible Version](http://www.washington.edu/accesscomputing/AU/before.html)
#### Element path
li:nth-child(2) > a[href="\#"]
#### Snippet
`<a href="#">Academics</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.52 (foreground color: #8a94a8, background color: #e7ecd8, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 91.0.4472.101
This accessibility issue was found using Accessibility Insights for Web 2.27.0 (axe-core 4.2.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
